api/common/network.go

```diff
@@ -191,6 +205,7 @@ func interfaceToNetworkConfig(nic net.Interface, nicType corenetwork.InterfaceTy
 		InterfaceType: string(nicType),
 		NoAutoStart:   !isUp,
 		Disabled:      !isUp,
+		NetworkOrigin: params.NetworkOrigin(networkOrigin),
```

Comment:
Machiner and containers use to make the source of the observable network config.